src/api/Item.js

```diff
@@ -200,7 +201,12 @@ class Item extends Base {
             .delete({ url })
             .then(this.deleteSuccessHandler)
             .catch((e: $AxiosError<any>) => {
-                this.errorHandler(e);
+                const { response } = e;
+                if (response?.status === HTTP_STATUS_CODE_NOT_FOUND) {
+                    this.deleteSuccessHandler();
```

Comment:
could a 404 happen if the user loses access to the content instead of it being deleted? in that case, it wouldn't be a success right?

Reply:
for files that were deleted by other users, it seems more accurate to call `this.errorHandler(e);` since the deletion wasn't a result of this api call